### ISSUE TITLE
Setting different FDR values?

### ISSUE THREAD

**Issue by Dmorgen** [smith-chem-wisc/MetaMorpheus#1743]
Is it possible to set different FDR values (peptide/protein)?

Thanks!
D.

**Comment by rmillikin** [comment]
can you clarify? the q-value is reported in one of the text columns and you can go down the list to whatever you want. there is a setting in the Output Options of the search task to filter output to a given q-value.

only PSMs <1% FDR are used for parsimony. same with quantification. do you want to change that?

**Comment by Dmorgen** [comment]
yes - I'd like to try several FDR setting for certain searches I'm running (various proteases and native peptides). is that possible?

**Comment by rmillikin** [comment]
this is not a currently user-alterable setting, but it should be pretty easy to make it one. maybe not the next release, but probably the one after that
